code/API_definitions/location.yaml

```diff
@@ -164,7 +169,30 @@ components:
       format: double
       minimum: -180
       maximum: 180
-      example: 7.10066        
+      example: 7.10066      
+    PostalCode:
+      type: object
+      description: Delivery area associated to a postal code. Each country defines its own postal code system.
+      required:
+        - type
+        - postalCode
+      properties:
+        type:
+          type: string
+          enum:
+            - PostalCode
+          description:  Type of this area.
+        postalCode:
+          type: string
+          description: Postal Code (aka Zip code or Postcode), according to the local format.
+        countryCode:
+          type: string
+          pattern: "^[A-Z]{2}$"
+          description: ISO 3166-1 alpha-2 code for the country. Uppercase. If not specified, each implementation may assume a default value.
```

Comment:
2 issues with countryCode:
- in the DeviceStatus API it means the MCC code and countryName means the ISO 3166 code
- I think it would be better to make this parameter mandatory, otherwise the API call without this parameter will have different results for the different provider implementations

Reply:
It is good to align terminology among APIs. To me this is more a "code" than a "name", but if the other term is agreed and consolidated in the Commonalities Glossary I'm open to align it.
I'm also open to make the property required.